grpcurl.go

```diff
@@ -161,6 +163,37 @@ func MetadataFromHeaders(headers []string) metadata.MD {
 	return md
 }
 
+/* Expands environmental variables contained in the header string
```

Comment:
Note I went with this comment format, as golint was complaining.

Reply:
Please stick with `//`, for consistency in this file.

To appease go lint, you need the comment to start with the named of the doc'ed thing, in this case "Expand Headers":

```
// ExpandHeaders expands environment variable references contained in the string.
// etc...
```

Reply:
Done.